integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -57,6 +59,9 @@ public class CatalogsPageTest extends AbstractWebIT {
   protected static String mysqlUri = "jdbc:mysql://127.0.0.1";
   protected static String postgresqlUri = "jdbc:postgresql://127.0.0.1";
 
+  private static final long MAX_WAIT_IN_SECONDS = 5;
+  private static final long WAIT_INTERVAL_IN_SECONDS = 1;
```

Comment:
I would suggest reducing the value of the wait interval, maybe 500ms or so is more suitable.

Reply:
'WAIT_INTERVAL_IN_SECONDS' is refer to other it test for this value.